src/main.rs

```diff
@@ -146,13 +145,13 @@ struct Args {
 
 /// Set up display pager
 #[cfg(not(target_os = "windows"))]
-fn configure_pager() {
-    Pager::with_default_pager(PAGER_COMMAND).setup();
+fn configure_pager(_: bool) {
+    Pager::with_default_pager("less -R").setup();
 }
 
 #[cfg(target_os = "windows")]
-fn configure_pager() {
-    eprintln!("Warning: -p / --pager flag not available on Windows!");
+fn configure_pager(enable_styles: bool) {
```

Comment:
This one (and the general pattern of passing things like `Args`, `quietly` or `enable_styles` to functions) annoys me a bit. It's convoluted, configuring a pager shouldn't know about printing style. I think we would be better of with a `OutputManager` or so that gets passed as a whole and has `print_error` and `print_warning` as methods. This could be expanded to a struct `ProgramState` or similar that also has the cli arguments and can tell callers things about them - without them having to look at the flags themselves. (I am not sure whether the latter would work out to be useful in practice though.) This is all, of course, nothing for this PR but something that could be done sometime after the next release.

Reply:
> This one (and the general pattern of passing things like Args, quietly or enable_styles to functions) annoys me a bit. It's convoluted, configuring a pager shouldn't know about printing style.

I fully agree! But I didn't really know how to easily solve it in a better way, and I think the new code is still better than the old.

Reply:
Agreed, let's just leave the comment open and merge this :)